pkgs/top-level/python-packages.nix

```diff
@@ -7630,7 +7630,7 @@ in {
 
   rope = callPackage ../development/python-modules/rope { };
 
-  ROPGadget = callPackage ../development/python-modules/ROPGadget { };
+  ropgadget = callPackage ../development/python-modules/ropgadget { };
```

Comment:
Should we move this out of pythonPackages? It looks like an end user program.

Reply:
Would definitely make sense to expose the script part.